src/app/main.cpp

```diff
@@ -208,14 +208,17 @@ int main(int argc, char *argv[])
 #endif
 
 #if defined(Q_OS_MAC)
-{
-    // Since Apple made difficult for users to set PATH, we set here for convenience.
-    // Users are supposed to install Homebrew Python for search function.
-    // For more info see issue #5571.
-    QByteArray path = "/usr/local/bin:";
-    path += qgetenv("PATH");
-    qputenv("PATH", path.constData());
-}
+        {
+            // Since Apple made difficult for users to set PATH, we set here for convenience.
+            // Users are supposed to install Homebrew Python for search function.
+            // For more info see issue #5571.
+            QByteArray path = "/usr/local/bin:";
+            path += qgetenv("PATH");
+            qputenv("PATH", path.constData());
+        }
```

Comment:
please do us a favor if you don't mind: remove the curly braces here, it isn't really needed.